### PR TITLE
Change to with-no-violation if no-violation is preceded by the number

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
Ref: https://github.com/stolostron/console/pull/3731#discussion_r1715434401

### CODE REVIEW

File: frontend/src/routes/Home/Overview/OverviewPage.test.tsx [hunk 1]
```diff
@@ -1115,8 +1115,8 @@ it('should render overview page with expected data', async () => {
 
   // Check Cluster violations chart rendered
   await waitFor(() => expect(getAllByText('Cluster violations')).toHaveLength(2))
-  await waitFor(() => expect(getByText('1 No violations')).toBeTruthy())
-  await waitFor(() => expect(getByText('1 Violations')).toBeTruthy())
+  await waitFor(() => expect(getByText('1 With no violations')).toBeTruthy())
```
Comment (mprahl) [inline frontend/src/routes/Home/Overview/OverviewPage.test.tsx:1118]: This should be lowercase.

File: frontend/src/routes/Home/Overview/OverviewPage.test.tsx [hunk 2]
```diff
@@ -1115,8 +1115,8 @@ it('should render overview page with expected data', async () => {
...
+  await waitFor(() => expect(getByText('1 With violations')).toBeTruthy())
```
Comment (mprahl) [inline frontend/src/routes/Home/Overview/OverviewPage.test.tsx:1119]: This should be lowercase.

File: frontend/src/routes/Home/Overview/OverviewPage.test.tsx (hunk 1 above)
Comment (yiraeChristineKim) [reply frontend/src/routes/Home/Overview/OverviewPage.test.tsx:1118]: This page's other badges are all Capital

Comment (mprahl) [reply frontend/src/routes/Home/Overview/OverviewPage.test.tsx:1118]: Ah okay, that should be fine then.

**Review by mprahl — APPROVED** [review]

**Review by dhaiducek — APPROVED** [review]

**Pull request merged**